### PR TITLE
Adds the clown bomb

### PR DESCRIPTION
**Pull request by Veritius**
## About the PR
Adds the clown bomb, a bomb that spawns 20 feral clowns on detonation.
Feral clowns will honk anything nearby relentlessly. They don't do any damage.

This PR also changes ClusterBombComponent and ClusterBombSystem to have an option to not use ThrowingSystem and use physics impulses instead, for moving KinematicControllers.

**Screenshots**
image

**Changelog**

:cl:
- add: Adds the Clown Bomb, an admeme item.

### CODE REVIEW

**Comment by metalgearsloth** [comment]
> Every single clown that ClusterBombSystem tries to spawn gives a warning (as you can't throw a KinematicController.) As an admeme item, this is tolerable, but if it's ever going to be made user accessible in normal rounds, that'll have to be fixed.

Need to do what atmos does and overwrite it temporarily (wyci).

File: Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml [hunk 1]
```diff
@@ -184,3 +185,39 @@
...
+        - state: icon
+          map: ["enum.TriggerVisualLayers.Base"]
+    - type: Item
+      size: 5
+    - type: OnUseTimerTrigger
+      delay: 10
+    - type: ClusterGrenade
+      fillPrototype: MobHumanFeralClown
+      directImpulse: true
+      maxGrenadesCount: 20
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml:207]: this is WAY too many clowns, this is like 300 ents getting spawned in. 5 max.

File: Content.Server/Explosion/Components/ClusterGrenadeComponent.cs [hunk 2]
```diff
@@ -43,6 +43,12 @@ public sealed class ClusterGrenadeComponent : Component
         [ViewVariables(VVAccess.ReadWrite)] [DataField("distance")]
         public float ThrowDistance = 50;
 
+        /// <summary>
+        ///     Use physics impulse instead of
+        /// </summary>
+        [ViewVariables(VVAccess.ReadWrite)] [DataField("directImpulse")]
+        public bool UsePhysics = false;
```
Comment (metalgearsloth) [inline Content.Server/Explosion/Components/ClusterGrenadeComponent.cs:50]: Property doesn't seem to do anything as they don't get launched anyway. Better off just having them actually throw? Just use a timer on the system to accumulate when to stop making them dynamic bodytypes and make them KC again.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
Still lagging my game real hard and not yeeting when I just tried it.

**Comment by Veritius** [comment]
Yeah I dunno why the yeeting isn't working. I was a bit busy on making the flavor text pr so I wasn't working on this. I'll just mark it as draft until I fix that.

**Comment by Veritius** [comment]
kinda given up on this one, just tag it with Derelict or smth

**Pull request closed without merging**